ceph-dashboard-pull-requests/config/definitions/ceph-dashboard-pull-requests.yml

```diff
@@ -62,3 +62,21 @@
           !include-raw:
             - ../../setup/setup
       - shell: "cd src/pybind/mgr/dashboard; timeout 7200 ./run-frontend-e2e-tests.sh"
+
+    publishers:
+      - cobertura:
+          report-file: "src/pybind/mgr/dashboard/frontend/coverage/cobertura-coverage.xml"
+          only-stable: "true"
+          health-auto-update: "false"
+          stability-auto-update: "false"
+          zoom-coverage-chart: "true"
+          source-encoding: "Big5"
```

Comment:
the encoding looks strange Big5 is for encoding of traditional Chinese in general. are you sure about this?